### ISSUE TITLE
SGCN/Season combinations in SGCN point that are not SGCNs

### ISSUE THREAD

**Issue by ChristopherTracey** [PNHP/COA#68]
As part of the table cleanup and validation issues I was exploring today, I discovered that there were a number of point records for SGCN birds for seasons that they are not considered SGCN. Here's the list of ones that were in the dataset:

- ABNCA02010_m           Pied-billed Grebe
- ABNGA01020_m           Least bittern
- ABNGA02010_m           American bittern
- ABNGA04040_m           great egret
- ABNJB10010_m            green winged teal
- ABNKC01010_m           osprey
- ABNKC10010_m           bald eagle
- ABNKC11010_m           northern harrier
- ABNME05030_m          Virginia rail
- ABNME08020_m          sora
- ABNME14020_m          American coot
- ABPAE32010_m           olive sided flycatcher
- ABPBG10010_m          sedge wren
- ABPBG10020_m          marsh wren
- ABPBJ18100_m           swainsons thrush
- ABPBX03230_m          blackpoll warbler
- ABPBX07010_m          prot warbler
- AMACC03020_b         tricolorerd bat
- IMBIV14060_y             Elliptio complanata, this isn't a SGCN--deleting for now.

I believe these largely came in from the eBird dataset (except the bat) and we didn't filter them out. We should check the eBird scripts and then check the derived files. 

@dyeany can you confirm that these are not SGCN?

**Comment by dyeany** [comment]
Yes, these species-season combinations do not qualify as SGCN. They should be filtered out as we had planned. I think this falls into the same category of data we discussed yesterday - where all the SGCN High Occ Prob data will have been filtered out as such. This criterion should be included.

**Comment by ChristopherTracey** [comment]
I wonder if we meant to do this with the ebird script--we could probably just add a filter for this for the future updates.

I deleted these from the current draft version of the 50million layer.
